ern-core/src/MavenUtils.ts

```diff
@@ -42,7 +42,15 @@ export default class MavenUtils {
     } else if (repoType === 'http') {
       mavenUser = mavenUser ? `'${mavenUser}'` : 'mavenUser'
       mavenPassword = mavenPassword ? `'${mavenPassword}'` : 'mavenPassword'
-      return `repository(url: "${mavenRepositoryUrl}") { authentication(userName: ${mavenUser}, password: ${mavenPassword}) }`
+      let authUrl = ''
```

Comment:
This variable is not properly named, there is no url involved here. `authBlock` or `authObject` might be more appropriate.